.github/workflows/zulip-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Zulip Tests
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
3.5 is already EOL and should be removed.

Reply:
okay sure

Reply:
We can also add 3.9 into the mix now.

Reply:
okay, yeah I hope that will be good.